lib/config_file.js

```diff
@@ -3,6 +3,7 @@ var readFileSync = require('fs').readFileSync;
 
 var configFiles = [ '.kibana-plugin-helpers.json', '.kibana-plugin-helpers.dev.json' ];
 var configCache = {};
+var KIBANA_ROOT_OVERRIDE = process.env.KIBANA_ROOT ? resolve(process.env.KIBANA_ROOT) : null;
```

Comment:
Changing environment variable around is a lot easier than writing files to the filesystem. But I'm not actually using it right now, just feels like a feature that should exist... I can remove it